clang/include/clang/ExtractAPI/DeclarationFragments.h

```diff
@@ -57,7 +57,7 @@ class DeclarationFragments {
     Keyword,
     Attribute,
     NumberLiteral,
-    StringLiteral,
+    StringRef,
```

Comment:
This is an unnecessary find-and-replace. This enum constant specifically references a string literal, not the `StringLiteral` type.

Reply:
good catch! i'll pore over the actual diff line by line to catch more things like this, but I wanted to first see if this PR would be shot down at a higher level :-)

Reply:
Generally a lot of clang changes are wrong. There's a StringLiteral AST node, without relation to StringRef...

Reply:
Thanks for the heads up -- will fix one by one.

Also, note to self:  let's leep `StringLiteral` around as deprecated to give downstreams a smoother transition.